smart_meter_texas/const.py

```diff
@@ -1,23 +1,28 @@
 import datetime
 
-BASE_HOSTNAME = "smartmetertexas.com"
-BASE_URL = "https://" + BASE_HOSTNAME + "/"
-BASE_ENDPOINT = BASE_URL + "api"
-AUTH_ENDPOINT = "/user/authenticate"
-DASHBOARD_ENDPOINT = "/dashboard"
-LATEST_OD_READ_ENDPOINT = "/usage/latestodrread"
-METER_ENDPOINT = "/meter"
-OD_READ_ENDPOINT = "/ondemandread"
-INTERVAL_SYNCH = "/adhoc/intervalsynch"
+BASE_HOSTNAME = "www.smartmetertexas.com"
+BASE_URL = "https://" + BASE_HOSTNAME
+BASE_ENDPOINT = BASE_URL 
+AUTH_ENDPOINT = "/commonapi/user/authenticate"
```

Comment:
Why change so much of the logic as part of this diff?

We could instead do here:
`AUTH_ENDPOINT = BASE_URL + "/commonapi/user/authenticate"`

That way everything else would be untouched.

Reply:
Makes sense